### PR TITLE
Update main.html

### PR DESCRIPTION
**Pull request by tushar-temp**

### CODE REVIEW

**Comment by soulee-dev** [comment]
Thank you for contributing for our project.
But seems like this commit is not that really useful for our repo.

@phg98 How do you think this commit?

File: theme/main.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
-
+<!-- HTML file to show on browser -->
```
Comment (soulee-dev) [inline theme/main.html:2]: 이런식의 단순 주석도 받아줘야 할까요?
계정을 살펴보니 핵토버 관련 프로젝트만 기여하는 스팸 기여자인것 같아서요...
@phg98

Comment (phg98) [reply theme/main.html:2]: 받아주면 안되겠습니다.
핵토버 규정에도 어긋나는듯요.

Comment (soulee-dev) [reply theme/main.html:2]: 그러면 간단하게 이유 영어로 적어주고 Close 하겠습니다!

**Comment by soulee-dev** [comment]
Thank you for your contribution to Hacktoberfest Korea. Unfortunately, the quality of your PR doesn't meet the required standards set by Hacktoberfest. Please refine and resubmit. We appreciate your effort and look forward to a revised submission.

**Pull request closed without merging**